mono/dis/push-pop.h

```diff
@@ -1,3 +1,5 @@
+#error This file is not used.
```

Comment:
just remove it?

Reply:
It demonstrates some nice patterns..could  also not change  it..